guides/release/testing/testing-components.md

```diff
@@ -156,23 +155,23 @@ Imagine you have the following component that changes its title when a button is
 > component magic-title`.
 
 ```javascript {data-filename="app/components/magic-title.js"}
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  title: 'Hello World',
+export default class MagicTitleComponent extends Component {
+  title = 'Hello World';
 
-  actions: {
-    updateTitle() {
-      this.set('title', 'This is Magic');
-    }
-  }
-});
+  @action
+  updateTitle() {
+    this.set('title', 'This is Magic');
+  };
```

Comment:
```suggestion
  }
```
I think method definitions and the class exports do not require a semicolon so we should remove them. You can see the bulk of the examples in https://github.com/ember-learn/guides-source/blob/octane/guides/release/upgrading/editions.md#native-classes too for reference.

Reply:
Right. I have been remiss in making sure I do this everywhere.